### PR TITLE
Setting concurrent runs of workflows in adapters.

### PR DESCRIPTION
**Pull request by sagarkapare**
Tested the fix on standalone as well as cluster.

### CODE REVIEW

**Comment by sagarkapare** [comment]
Build is running: https://builds.cask.co/browse/CDAP-RBT111-1

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/adapter/AdapterServiceTests.java [hunk 1]
```diff
@@ -76,7 +82,10 @@ public void testAdapters() throws Exception {
 
     // Create Adapter
     adapterService.createAdapter(namespaceId, adapterSpecification);
-
+    PreferencesStore preferencesStore = getInjector().getInstance(PreferencesStore.class);
+    Map<String, String> prop = preferencesStore.getResolvedProperties(namespaceId,
+                                                                             adapterSpecification.getType());
```
Comment (anwar6953) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/adapter/AdapterServiceTests.java]: the indentation on the second line is a bit off

**Comment by anwar6953** [comment]
LGTM

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/adapter/AdapterServiceTests.java (hunk 1 above)
Comment (sagarkapare) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/adapter/AdapterServiceTests.java]: Modified.

**Pull request merged**